packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -28,6 +28,8 @@ export async function onFormSubmission(
     // It isn't an eventType webhook
     eventTypeId: -1,
     triggerEvent: WebhookTriggerEvents.FORM_SUBMITTED,
+    // When team routing forms are implemented, we need to make sure to add the teamId here
+    teamId: -1,
```

Comment:
Can we use teamId: null? instead of -1?

Reply:
Changed it to 0 instead, as everywhere else we also fallback to 0. `getWebhooks` does not allow a `null` value, we would need to change that then